pkgs/tools/misc/grub/pvgrub_image/configs/grub.cfg

```diff
@@ -0,0 +1,9 @@
+if search -s -f /boot/grub/grub.cfg ; then
+	echo "Reading (${root})/boot/grub/grub.cfg"
```

Comment:
My understanding is that it is a convenience feature, which allows you to have a separate `/boot` partition and still use the same pre-built pvgrub image, as explained on the [Xen project wiki](https://wiki.xen.org/wiki/PvGrub2):
>  Bootstrapping into this more complex grub.cfg allows us to cope with guests which have a separate /boot partition by searching for a file named /boot/grub/grub.cfg or /grub/grub.cfg on any partition. Note that ${root} is set by the search command and should be entered literally, not substituted while creating this file.

Since the Xen project wiki is advocating this solution, I would recommend keeping it.